.github/workflows/ci.yaml

```diff
@@ -23,7 +31,7 @@ jobs:
         cache: 'pip' #Note that pip is for the tox level. Poetry is still used for installing the specific environments (tox.ini)
 
     - name: Install tox
-      run: pip install tox
+      run: pip install tox==4.15.0
```

Comment:
It would be nice if you explained why this change is needed.

Reply:
The idea is to keep the pipeline as consistent as possible between runs, and to make sure it is the developer's conscious decision to use the next version during the workflow.

Reply:
I agree on consistency, I was curious whether there was a reason for this specific version.

Reply:
Ah oke, this is just the current version, that's why.